test/libdnf5/conf/test_conf.cpp

```diff
@@ -71,3 +71,57 @@ void ConfTest::test_config_pkg_gpgcheck() {
     CPPUNIT_ASSERT_EQUAL(&config_repo.get_pkg_gpgcheck_option(), &config_repo.get_gpgcheck_option());
 #pragma GCC diagnostic pop
 }
+
+libdnf5::ConfigMain copy_construct_config_main(const libdnf5::ConfigMain & config) {
```

Comment:
Move the functions `copy_construct_config_main`, `assign_config_main`, and `test_copy_config_main` to anonymous namespace:

```
namespace {
... functions ...
}
```
We do not want to pollute the global namespace. All tests files are linked together. I remember problems where a class with the same name was defined in multiple test files within the global namespace.